merkle-tree/src/db/common/adminDbConnection.js

```diff
@@ -10,6 +10,11 @@ else url = `mongodb://${host}:${port}`;
 dbConnections.admin = mongoose.createConnection(`${url}/${databaseName}`, {
   useNewUrlParser: true,
   useCreateIndex: true,
+  authSource: "admin",
+  user: "admin",
+  pass: "admin",
+  useUnifiedTopology: true,
+  useFindAndModify: false 
```

Comment:
This could be an issue for cloud dbs (such as MongoDB Atlas). I'm going to try and find a more universal way of making this work.